src/themes/default/elements/icon.overrides

```diff
@@ -400,6 +446,10 @@ i.icon.unmute:before { content: "\f130"; }
 i.icon.mute:before { content: "\f131"; }
 i.icon.video.play:before { content: "\f144"; }
 i.icon.closed.captioning:before { content: "\f20a"; }
+i.icon.pause.circle:before { content: "\f28b"; }
+i.icon.pause.circle.outline:before { content: "\f28c"; }
+i.icon.stop.circle:before { content: "\f28d"; }
+i.icon.stop.circle.outline:before { content: "\f28e"; }
```

Comment:
New icons for the Audio category: `pause circle`, `pause circle outline`, `stop circle`, `stop circle outline`